webapp/cube/views.py

```diff
@@ -45,14 +46,22 @@
 )
 
 
+def is_authorized(user):
```

Comment:
New decorator?

`@canonical_employees` or something like this

Reply:
could be, this will be removed after as while, so I thought I would just keep it simple

Reply:
ok why not.